src/checkMinRequiredVersion.ts

```diff
@@ -0,0 +1,20 @@
+import * as semver from "semver";
+
+import { configstore } from "./configstore";
```

Comment:
What's the workflow for updating configstore?

Reply:
@mbleigh knows more about this than I do - it involves updating an RTDB node on a project we own. When we want to set this min version, he should be able to help us get it set up.